extras/craft/gulprev/twigextensions/GulpRevTwigExtension.php

```diff
@@ -30,12 +30,20 @@ public function gulpRev($file)
 
         // If the file is not defined in the asset manifest
         // just return the original string
-        $path = $file;
+        $path          = $file;
+        $manifest_path = 'rev-manifest.json';
```

Comment:
Can you see if `$craftPath` works here? It's defined in `public/index.php`. Basically, we should be more explicit about the path. Then we can do `$manifest_path = $craftPath.'/public/rev-manifest.json';`.